command/v6/remove_network_policy_command.go

```diff
@@ -52,11 +58,42 @@ func (cmd *RemoveNetworkPolicyCommand) Setup(config command.Config, ui command.U
 }
 
 func (cmd RemoveNetworkPolicyCommand) Execute(args []string) error {
+	switch {
```

Comment:
A `switch` statement feels odd here, given that there's only one case.

Reply:
🤔